mis_builder/models/mis_report_instance.py

```diff
@@ -307,11 +307,22 @@ def _onchange_source(self):
     def _get_filter_domain_from_context(self):
         filters = []
         mis_report_filters = self.env.context.get('mis_report_filters', {})
-        for filter_name, values in mis_report_filters.items():
-            if values:
-                value = values.get('value')
-                operator = values.get('operator', '=')
-                filters.append((filter_name, operator, value))
+        for filter_name, domain in mis_report_filters.items():
+            if domain:
+                value = domain.get('value')
+                operator = domain.get('operator')
+                if not value:
+                    continue
```

Comment:
Is this continue important? Testing for "is not set" (= False) could make sense in some situations.